sirepo/template/radia.py

```diff
@@ -27,11 +28,59 @@
 import sirepo.util
 import time
 
+class DataRequester(template_common.DataRequesterBase):
+    """Fetch data from the template either for download or to use on the client
+
+
+    """
+
+    def __init__(self):
+        """
+        Args:
+            xxx (xxx):
+           XXX
+        """
+        self.appExporter = Exporter()
+
+    def export(self, sim):
```

Comment:
This should not be an instance. Use a class method so that  you dont' need to cache the instance.

Reply:
dc82708